srcpkgs/NetworkManager-l2tp/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'NetworkManager-l2tp'
+pkgname=NetworkManager-l2tp
+version=1.8.2
+revision=1
+build_style=gnu-configure
+configure_args="--runstatedir=/run"
+hostmakedepends="pkg-config intltool glib-devel"
+makedepends="gtk+3-devel libglib-devel libsecret-devel network-manager-applet-devel
+libressl-devel nss-devel ppp-devel"
+depends="strongswan xl2tpd ppp"
+short_desc="NetworkManager VPN plugin for L2TP/IPsec"
+maintainer="Douglas Kosovic <doug@uq.edu.au>"
```

Comment:
> I was just trying to avoid being a maintainer of a NetworkManager-l2tp package for another Linux distro on top of the ones I already maintain. I don't have anything against a PR, I just wanted to avoid being the maintainer.

If this is still true, feel free to have the package as Orphaned, and update it when/if you have time, rather than being the "goto person" for this template.

Reply:
happy to carry on for a while, can orphan it later.